lib/routes/home/widgets/payments_list/widgets/payment_details_sheet/payment_details_sheet.dart

```diff
@@ -229,6 +234,13 @@ class PaymentDetailsSheet extends StatelessWidget {
                           unblindingData: paymentData.unblindingData,
                         ),
                       ],
+                      if (claimTxId.isNotEmpty) ...<Widget>[
+                        PaymentDetailsSheetTxId(
+                          txId: claimTxId,
+                          unblindingData: paymentData.unblindingData,
+                          isBtcTx: true,
```

Comment:
This depends on the direction of the payment. When sending its a Bitcoin tx, receiving its a Liquid tx

Reply:
Thanks.

Done. 6e7ecc4435d1e57e96c94b1a18c6138526783e6c